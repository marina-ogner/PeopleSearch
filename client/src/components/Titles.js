import React from "react";

class Titles extends React.Component {
    render() {
        return (
            <div>
                <h1>Welcome to Versium People Search</h1>
            </div>
        );
    }
};

export default Titles;