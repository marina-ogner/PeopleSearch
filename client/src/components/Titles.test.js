import React from 'react';
import Titles from './Titles';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('Titles component should render as expected', () => {
    const component = shallow(<Titles />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
})
