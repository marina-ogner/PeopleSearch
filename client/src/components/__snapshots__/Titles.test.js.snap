// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titles component should render as expected 1`] = `
<div>
  <h1>
    Welcome to Versium People Search
  </h1>
</div>
`;
